src/pages/markets/components/sections/_other-markets.tsx

```diff
@@ -153,6 +154,9 @@ const StyledFlex = styled(Flex)`
 
     &:hover {
         box-shadow: 0 4px 8px 0 rgba(14, 14, 14, 0.1);
+        width: 282px;
```

Comment:
We can remove the `width` here, it's thesame with the original state

Reply:
okay